### DISCUSSION TITLE
ClickSend Notification Channel

### DISCUSSION THREAD

**Discussion by webrobert** [laravel/framework#45806] [Q&A] [answered]
I’d like to offer the addition of clicksend as a sms notification channel. The contribution text said be willing to work on any additions. I’ve authored the code based on the vonage channel.  It works I’ve already run it.  All that remains is to confirm the tests on the package. 

My question is who do I send it to, and how do I actually run tests, haha.  It uses orchestra. It’s my first package. 

https://github.com/webrobert/clicksend-notification-channel

**Comment by henzeb** [comment, marked as answer]
Nice, maybe you could also send it here. https://laravel-notification-channels.com/about/

you run tests with phpunit. `vendor/bin/phpunit` would do it.

**Reply by webrobert** [reply]
Yeah, I totally missed https://laravel-notification-channels.com/ I just downloaded the skeleton, I'm going to re package it to submit there. Thank you!
